JS/Lesson31/saffet/my-app/src/components/ListItem/index.jsx

```diff
@@ -1,8 +1,18 @@
+import { useState } from 'react';
 import './styles.css';
 
 export const ListItem = ({ id, title, handleDelete }) => {
+
+	const [completed, setCompleted] = useState(false);
+
+	const handleClick = () => {
+		setCompleted(!completed);
+	}
+
+	const className = completed ? 'completed' : '' ;
+
 	return (
-		<li>
+		<li onClick={handleClick} className={className}>
```

Comment:
If you add a span around `title` and move the `className` to it, then only the text of the list item would be crosses when an item is checked :) 
```suggestion
		<li onClick={handleClick}>
		  <span className={className}>{title}</span>

```